ui-v2/tests/unit/search/filters/node-test.js

```diff
@@ -19,6 +20,7 @@ module('Unit | Search | Filter | node', function() {
     [
       {
         Node: 'name',
+        Address: '10.0.0.0',
       },
     ].forEach(function(item) {
       const actual = filter(item, {
```

Comment:
Good spot finding these tests and adding to them 👏  I think we should maybe add a brand new `test()` case here though, right now I don't think we are asserting that an `item` can be found by `Address`. Ooooh I'm wondering if our soon to be merged coverage reporting would help with this 🤔 

Anyhoo lets catch up later on this.